python/pyarrow/tests/test_fs.py

```diff
@@ -1321,22 +1321,23 @@ def test_s3fs_wrong_region():
     from pyarrow.fs import S3FileSystem
 
     # wrong region for bucket
-    fs = S3FileSystem(region='eu-north-1')
+    fs = S3FileSystem(region='eu-north-1', anonymous=True)
```

Comment:
I'd add a comment explaining why this is important. Do we have other tests s3 tests that could also need this?

Reply:
Done, and I found only one other test that uses a real AWS call, aptly named `test_s3_real_aws` but that already sets `anonymous=True`. All others appear to use the endpoint override / minio.